### ISSUE TITLE
"fig_zsh_redraw" when pasting into terminal

### ISSUE THREAD

**Issue by fwesss** [withfig/fig#449]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue

### Description:
When pasting text in the terminal then typing, I see two errors about fig_zsh_redraw in the terminal:

```bash
fig_zsh_redraw:zle:2: No handler installed for fd 14
fig_zsh_redraw:3: file descriptor 14 used by shell, not closed
```

https://user-images.githubusercontent.com/10150898/132774239-c83cd282-8885-4837-bcc3-ad83069fd892.mp4

### Environment

```bash
N/A
```

**Comment by mschrage** [comment]
I've run into this a few times as well. Tricky to reproduce, but definitely not a one off.

**Comment by mschrage** [comment]
Found a way to reliably reproduce:

Pasting `go mod init` into zsh and then pressing the spacebar.

**Comment by nasyxx** [comment]
I also met this issue when I pasted the content into zsh.

On iterm2.

**Comment by a-eid** [comment]
same issue.

**Comment by mschrage** [comment]
Hey @a-eid! This should be fixed in the next release (v1.0.54).
